core/src/mindustry/ui/dialogs/PlanetDialog.java

```diff
@@ -1160,7 +1160,7 @@ void playSelected(){
                 //free launch.
                 control.playSector(sector);
             }else{
-                CoreBlock block = from.planet.allowLaunchSchematics ? (from.info.bestCoreType instanceof CoreBlock b ? b : (CoreBlock)from.planet.defaultCore) : (CoreBlock)from.planet.defaultCore;
+                CoreBlock block = from.allowLaunchSchematics() ? (from.info.bestCoreType instanceof CoreBlock b ? b : (CoreBlock)from.planet.defaultCore) : (CoreBlock)from.planet.defaultCore;
```

Comment:
Okay but now you changed too many things, you're checking if the **from** sector allows launch schematics, which is not what you want, it should still be `sector.allowLaunchSchematics()`.